<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1267</width>
    <height>789</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Cgmes Explorer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/Icons/icons/roseta.svg</normaloff>:/Icons/icons/roseta.svg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="iconSize">
   <size>
    <width>48</width>
    <height>48</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="mainTabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="dataTabLayout">
          <attribute name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/Icons/icons/array.svg</normaloff>:/Icons/icons/array.svg</iconset>
          </attribute>
          <attribute name="title">
           <string>Data</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QTabWidget" name="modelTabWidget">
             <property name="tabPosition">
              <enum>QTabWidget::South</enum>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="modelTab">
              <attribute name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/Icons/icons/array.svg</normaloff>:/Icons/icons/array.svg</iconset>
              </attribute>
              <attribute name="title">
               <string>Model</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QSplitter" name="main_splitter">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <widget class="QFrame" name="frame_2">
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QFrame" name="frame_7">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>26</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::NoFrame</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_2">
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item>
                       <widget class="QLabel" name="modelTypeLabel">
                        <property name="text">
                         <string>...</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <spacer name="horizontalSpacer">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>529</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QListView" name="clasesListView"/>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QFrame" name="frame_6">
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <property name="leftMargin">
                    <number>1</number>
                   </property>
                   <property name="topMargin">
                    <number>1</number>
                   </property>
                   <property name="rightMargin">
                    <number>1</number>
                   </property>
                   <property name="bottomMargin">
                    <number>1</number>
                   </property>
                   <item>
                    <widget class="QFrame" name="frame_4">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::NoFrame</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout">
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item>
                       <widget class="QLineEdit" name="filterLineEdit"/>
                      </item>
                      <item>
                       <widget class="QComboBox" name="filterComboBox"/>
                      </item>
                      <item>
                       <widget class="QPushButton" name="filterButton">
                        <property name="text">
                         <string>Filter</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QTableView" name="propertiesTableView"/>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="loggerTab">
              <attribute name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/Icons/icons/data.svg</normaloff>:/Icons/icons/data.svg</iconset>
              </attribute>
              <attribute name="title">
               <string>Logger</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QTreeView" name="loggerTreeView"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="dbTab">
          <attribute name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/Icons/icons/data.svg</normaloff>:/Icons/icons/data.svg</iconset>
          </attribute>
          <attribute name="title">
           <string>Lookup Database</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QSplitter" name="lookup_db_splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QFrame" name="frame_3">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <property name="rightMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QFrame" name="frame_9">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QComboBox" name="availableDBsComboBox">
                    <property name="minimumSize">
                     <size>
                      <width>220</width>
                      <height>0</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>392</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QListView" name="dbTablesListView"/>
               </item>
              </layout>
             </widget>
             <widget class="QFrame" name="frame_5">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QTableView" name="dbTableView"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab">
          <attribute name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/Icons/icons/console.svg</normaloff>:/Icons/icons/console.svg</iconset>
          </attribute>
          <attribute name="title">
           <string>Python</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="consoleLayout"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="settingsTab">
          <attribute name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/Icons/icons/gear.svg</normaloff>:/Icons/icons/gear.svg</iconset>
          </attribute>
          <attribute name="title">
           <string>Settings</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QFrame" name="frame_8">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="1" colspan="2">
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Databases</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>937</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>539</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="progress_frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="1" column="0">
        <widget class="QPushButton" name="cancelButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Cancel process</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/Icons/icons/delete.svg</normaloff>:/Icons/icons/delete.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QProgressBar" name="progressBar">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>159</red>
               <green>159</green>
               <blue>159</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="PlaceholderText">
             <brush brushstyle="SolidPattern">
              <color alpha="128">
               <red>159</red>
               <green>159</green>
               <blue>159</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>159</red>
               <green>159</green>
               <blue>159</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="PlaceholderText">
             <brush brushstyle="SolidPattern">
              <color alpha="128">
               <red>159</red>
               <green>159</green>
               <blue>159</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>120</red>
               <green>120</green>
               <blue>120</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="PlaceholderText">
             <brush brushstyle="SolidPattern">
              <color alpha="128">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QProgressBar {
	border: 1px solid rgb(186, 189, 182);
    border-radius: 5px;
	text-align: center;
}
QProgressBar::chunk{
	background-color: rgb(0, 34, 43);
    color: rgb(255, 255, 255)
}
</string>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="progressLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1267</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_logs"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionGenerate_PSSe_look_up_database"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionNavigation_tree"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuActions"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/icons/loadc.svg</normaloff>:/Icons/icons/loadc.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionConvert_to_psse">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/icons/to_psse.svg</normaloff>:/Icons/icons/to_psse.svg</iconset>
   </property>
   <property name="text">
    <string>Convert to PSS/e</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T, Ctrl+P</string>
   </property>
  </action>
  <action name="actionConvert_to_CGMES">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/icons/to_cgmes.svg</normaloff>:/Icons/icons/to_cgmes.svg</iconset>
   </property>
   <property name="text">
    <string>Convert to CGMES</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T, Ctrl+C</string>
   </property>
  </action>
  <action name="actionConvert_to_Roseta">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/icons/to_roseta.svg</normaloff>:/Icons/icons/to_roseta.svg</iconset>
   </property>
   <property name="text">
    <string>Convert to Roseta</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T, Ctrl+R</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/icons/savec.svg</normaloff>:/Icons/icons/savec.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save grid, the format will be propted</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_logs">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/icons/save.svg</normaloff>:/Icons/icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>Save logs</string>
   </property>
  </action>
  <action name="actionNavigation_tree">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/icons/tree.svg</normaloff>:/Icons/icons/tree.svg</iconset>
   </property>
   <property name="text">
    <string>Navigation tree</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/icons/new2c.svg</normaloff>:/Icons/icons/new2c.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/icons/new2.svg</normaloff>:/Icons/icons/new2.svg</iconset>
   </property>
   <property name="text">
    <string>Documentation</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/icons/roseta.svg</normaloff>:/Icons/icons/roseta.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionProcess_topology">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/icons/schematic.svg</normaloff>:/Icons/icons/schematic.svg</iconset>
   </property>
   <property name="text">
    <string>Process topology</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionRun_power_flow">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/icons/automatic_layout.svg</normaloff>:/Icons/icons/automatic_layout.svg</iconset>
   </property>
   <property name="text">
    <string>Run power flow</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionGenerate_PSSe_look_up_database">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/icons/import_models.svg</normaloff>:/Icons/icons/import_models.svg</iconset>
   </property>
   <property name="text">
    <string>Generate PSSe look-up database</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
